sql/core/src/test/resources/sql-tests/inputs/decimalArithmeticOperations.sql

```diff
@@ -49,7 +49,6 @@ select 1e35 / 0.1;
 
 -- arithmetic operations causing a precision loss are truncated
 select 123456789123456789.1234567890 * 1.123456789123456789;
-select 0.001 / 9876543210987654321098765432109876543.2
```

Comment:
I think it is missing a `;` before...

Reply:
yes, unfortunately I missed it somehow previously...